app/views/static_pages/home.html.erb

```diff
@@ -2,7 +2,8 @@
   <div class="container">
     <div class="objects-grids">
       <% @articles.each do |article| %>
-        <div class="col-md-4 objects-grid">
+        <% unless article.private? %>
```

Comment:
I think that this validation works better in controller

Reply:
Agree, with a filter or where